scripts/west

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
```

Comment:
is the path to `scripts/` already added to the `PATH`?

Reply:
Yes, this is done in zephyr_env.sh:

https://github.com/zephyrproject-rtos/zephyr/blob/master/zephyr-env.sh#L55

Note that the Windows launcher uses a DOS macro instead.

Reply:
Right, I saw that now yes. Had never heard of `doskey` even though I used DOS extensively back in the day.